clock_exercise/clockmanager.cpp

```diff
@@ -1,11 +1,17 @@
 #include <QTime>
 #include<QDebug>
+#include <QtGlobal>
 #include "clockmanager.h"
+#include "notifierofchange.h"
 
-ClockManager::ClockManager(QObject *parent) : QObject(parent), m_timerForRefresh(this)
+//int m_timerHoursOnStartTimer;
+//int m_timerMinutesOnStartTimer;
+
+ClockManager::ClockManager(QObject *parent) : QObject(parent), m_timerMinutes(0), m_timerHours(0), m_timerIsInPlay(false), m_timerForRefresh(this), m_timerCurrentSeconds(0)
```

Comment:
In questo caso vogliamo inizializzare il timer con `m_timerForRefresh(nullptr)`, dato che lo abbiamo come variabile membro, altrimenti Qt cerchera di chiamare la delete su questo oggetto.

Reply:
Direi che non c'è da inizializzare nulla visto che è una variabile membro.